frontend/components/dashboard/DashboardTransactionsRow.js

```diff
@@ -1,8 +1,54 @@
+import mustache from "mustache";
 import { Row, Col } from "react-bootstrap";
 
 import Timer from "../utils/Timer";
 import GetTransactionIcon from "../utils/GetTransactionIcon";
 
+const prettifyTransaction = (kind, args) => {
+  if (kind === "AddKey") {
+    try {
+      args = JSON.parse(args);
+    } catch (err) {
+      console.log(err);
+      return args;
+    }
+
+    return mustache.render(
```

Comment:
I have a concern regarding the need using mustache: https://github.com/nearprotocol/near-explorer/issues/32#issuecomment-514615945

Reply:
So you recommend to do it React way, by making separate components for each transaction type?

Reply:
If you look at the latest pr, you can see I had split it into a component per transaction type. But I had also used mustache as templating the string message inside `DataProvider`.

Reply:
I still don't understand the need for `mustache` over [in this particular case] JS templating strings.

I want to make a note, that I am not requiring the changes and avoiding `mustache`, I want to discuss and see if we really need it, like how this new dependency helps us.

Reply:
@frol So how do you recommend doing the same templating of strings (if we don't use `mustache`)?